src/sagemaker_containers/_mapping.py

```diff
@@ -165,16 +165,20 @@ def _is_property(self, _property):
         return isinstance(getattr(type(self), _property), property)
 
     def __getitem__(self, k):
+        """Placeholder docstring"""
```

Comment:
filling these in would be nice 🤣

Reply:
Planning on overhauling all docs and docstrings in https://github.com/aws/sagemaker-training-toolkit. These had somehow made it past `pylint` previously (unrelated file to the current PR), so just fixing violations for the purpose of this patch.